resolwe_bio/processes/chip_seq/macs14.yml

```diff
@@ -99,3 +105,6 @@
       then
         re-save-file neg_peaks ${NAME}_negative_peaks.xls
       fi
+
+      re-save species {{ treatment.species }}
```

Comment:
I suggest that you check in the beginning of the process that the treatment and control inputs have the same species and build. If not, raise an informative error.